disasm/disasm.go

```diff
@@ -230,17 +213,14 @@ func NewDisassembly(fn wasm.Function, module *wasm.Module) (*Disassembly, error)
 			// as well. To make sure that isInstrReachable
 			// returns the correct value, we don't push a new
 			// array to blockPolymorphicOps.
-			if !instr.Unreachable {
-				// Therefore, only push a new array if this instruction
-				// is reachable.
-				blockPolymorphicOps = append(blockPolymorphicOps, []int{})
-			}
+
+			// Therefore, only push a new array if this instruction
```

Comment:
I guess this comment block as well as the one above need some rewording in light of the control flow change that is being applied in this PR.